### ISSUE TITLE
[Question] Guideline to reproduce the code

### ISSUE THREAD

**Issue by nguyenvulong** [yzyouzhang/Empirical-Channel-CM#1]
Hello, thanks for sharing. I've read your one-class classification paper based on AM-Softmax and that was a very nice approach.

I would like to ask a general question regarding your repositories. It seems like to me the source code of this _"empirical paper"_ repo and the _"augmented channel paper"_ would **also** require data preparation using `Matlab`, is that correct? 

I did not see that instruction in the `README` section but I had a look at your first repo about _"one-class classification paper"_ so I assume that it's true. I also noticed that the baseline from [ASVspoof challenge](https://github.com/asvspoof-challenge) offers feature preprocessing in pure `Python`, is it possible to do the same with your repositories. The reason I ask this question is because `Matlab` is quite expensive to purchase, so i try to resort to `Python` if possible. 

Thanks a lot.

**Comment by yzyouzhang** [comment]
Hi,

Thanks for your interest in our work.
Actually, I've changed the data preparation process from MATLAB to Python since the empirical study. Please refer to `feature_extraction.py` in this repo. It includes the PyTorch code for LFCC. Our one-class work did use MATLAB code from the ASVspoof2019 organizers.

Please let me know if you have further questions. Thanks.

**Comment by nguyenvulong** [comment]
Thank you, if i have any questions while reproducing the experiment i will definitely ask you. Good day!
